fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/model/Generated.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * (C) Copyright IBM Corp. 2019
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.database.utils.model;
+
+public enum Generated {
+    ALWAYS, BY_DEFAULT;
+    
+    @Override
+    public String toString() {
+        switch (this) {
+        case BY_DEFAULT: return "BY DEFAULT";
```

Comment:
I realize this is a return, and naturally breaks.  maybe a comment? explaining the expected behavior ?

Reply:
added some javadoc